vilebot/src/test/java/vilebot/handlers/user/RemindMeTest.java

```diff
@@ -0,0 +1,43 @@
+package vilebot.handlers.user;
+
+import org.junit.Test;
+
+import com.oldterns.vilebot.handlers.user.RemindMe;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.Calendar;
+import java.util.concurrent.TimeUnit;
+
+import ca.szc.keratin.core.event.message.recieve.ReceivePrivmsg;
+
+public class RemindMeTest
```

Comment:
We should also potentially test that different user inputs are parsed correctly, e.g. `!remindme eat lunch 15`, `!remindme hello tomorrow`.